pkgs/os-specific/linux/rasdaemon/default.nix

```
@@ -0,0 +1,102 @@
+{ stdenv, lib, fetchurl, automake, autoreconfHook, coreutils, dmidecode
+, glibcLocales, hwdata, kmod, libtool, makeWrapper, sqlite, perl, doCheck ? true
+}:
+stdenv.mkDerivation rec {
+  name = "rasdaemon";
+  version = "0.6.4";
+  src = fetchurl {
+    url =
+      "http://www.infradead.org/~mchehab/rasdaemon/rasdaemon-${version}.tar.bz2";
```

```suggestion
      "https://www.infradead.org/~mchehab/rasdaemon/rasdaemon-${version}.tar.bz2";
```

Why not get it on pagure? There they already have version `0.6.4-2`